packages/js/src/executors/node/node.impl.ts

```diff
@@ -45,11 +56,13 @@ export async function* nodeExecutor(
     options.buildTarget,
     context.projectGraph
   );
+  const buildTargetOptions = project.data.targets[buildTarget.target];
```

Comment:
Ok, this name confuses me a bit, with `options.buildTargetOptions`. Let's change it to `buildTargetConfiguration`, since it's more accurate in any case.